[0.2.0] com.google.gson.JsonSyntaxException

I updated Javabrake from 0.1.6 to 0.2.0, changed `new Notifier(projectId, projectKey);` to `new Notifier(config);` and ran my application. I got this:

```
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:226)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:897)
	at com.google.gson.Gson.fromJson(Gson.java:846)
	at com.google.gson.Gson.fromJson(Gson.java:817)
	at io.airbrake.javabrake.PollTask.run(PollTask.java:65)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:386)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:215)
	... 7 common frames omitted
```

Any ideas?

--- a/src/main/java/PollTask.java
+++ b/src/main/java/PollTask.java
@@ -12,7 +12,7 @@ import okhttp3.HttpUrl;
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import com.google.gson.JsonParseException;
+import com.google.gson.JsonSyntaxException;
 
 class PollTask extends TimerTask {
   final private int projectId;
@@ -59,11 +59,18 @@ class PollTask extends TimerTask {
     String response = null;
     try {
       response = this.request();
-    } catch(IOException ex) {
+    } catch(IOException e) {
+      System.out.println(e.getMessage());
+      e.printStackTrace();
     }
 
-    RemoteConfigJSON json_data = gson.fromJson(response, RemoteConfigJSON.class);
-    this.data.merge(json_data);
+    try {
+      RemoteConfigJSON json_data = gson.fromJson(response, RemoteConfigJSON.class);
+      this.data.merge(json_data);
+    } catch(JsonSyntaxException e) {
+      System.out.println(e.getMessage());
+      e.printStackTrace();
+    }
 
     this.setErrorHost(this.data);
     this.processErrorNotifications(this.data);

